python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/monitor_sessions_default.schema.yml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  monitor_sessions_default_encapsulation_gre:
```

Comment:
```suggestion
  monitor_session_default_encapsulation_gre:
```

Reply:
make sure to update all file names accordingly.

Reply:
Fixed!